### PR TITLE
add mapset info to page title

### LINKED ISSUE #21: Change title of tab from "OMDB - Mapset" to the name of the map
image
It'd be easier to understand what the tab is you understand what I'm asking here

Ex: "The Quick Brown Fox - The Big Black Ratings" or similar

### PR DESCRIPTION
**Pull request by tilda**
simple set of changes. this probably works but i'm not sure since i don't have a php devenv handy, so please test for me.

closes #21
requires #34

### CODE REVIEW

**Comment by tilda** [comment]
btw ill probably submit more prs just on this file in general if you're fine with that, there's some refactoring i could get in as a "Modern JavaScript Programmer"

**Comment by apollo-dw** [comment]
hi, thanks for ur contribution 👀 i absolutely welcome any PRs so feel free to submit stuff and ill check it asap

i don't think this will immediately work though, `$PageTitle` is used in the `header.php` file, and `header.php` is included before it can be used. ofc it starts getting a little messy because `header.php` includes necessary connection stuff so it becomes a fault of my design, and the reason why i haven't been able to implement this yet...

i think we'll need to see `header.php` be split into two parts for us to do this, such that the top of this file would look something like:...

```php
include '../global.php'; // connections, functions, other necessary stuff

// mapset select statements here

$PageTile = "...";

include '../header.php'; // and then this would be mostly the visual front-end stuff, allowing us to set the title properly
```

**Comment by tilda** [comment]
ahh i see, this did look a little *too* simple... will come back to this in a bit then

File: mapset/index.php [hunk 1]
```diff
@@ -1,6 +1,14 @@
 <?php
 	$mapset_id = $_GET['mapset_id'] ?? -1;
 	require '../base.php';
+
+	$foundSet = false;
+	$result = $conn->query("SELECT * FROM `beatmaps` WHERE `SetID`='${mapset_id}' AND `mode`='0' ORDER BY `SR` DESC;");
+	$sampleRow = $result->fetch_assoc();
+	mysqli_data_seek($result, 0);
+
+	$PageTitle = htmlspecialchars($sampleRow['Title']) . " mapped by " . GetUserNameFromId($sampleRow['CreatorID'], $conn);
```
Comment (apollo-dw) [inline mapset/index.php:10]:
```suggestion
	$PageTitle = htmlspecialchars($sampleRow['Title']) . " by " . GetUserNameFromId($sampleRow['CreatorID'], $conn);
```

just to match the page's `<h1>` tag

Comment (tilda) [reply mapset/index.php]: yeah

**Pull request merged**